src/igtcloud/client/tools/upload_project.py

```diff
@@ -83,6 +87,80 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
                             f"files_skipped: {len(files_skipped)}")
 
 
+def upload_annotation_files_private_method(institutes, local_folder, max_workers_files):
+    # Annotation file upload
+    logger.info("Annotation File Uploading...")
```

Comment:
Log statement is a bit redundant, since the upload progress will be displayed anyways.

Reply:
Removed the logger